src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetPreviewTokenPage.tsx

```diff
@@ -92,51 +97,93 @@ export default function ReleaseApiDataSetPreviewTokenPage() {
       </Link>
       <LoadingSpinner loading={isLoadingDataSet || isLoadingPreviewTokenId}>
         <div className="govuk-grid-row">
-          <div className="govuk-grid-column-three-quarters">
+          <div className="govuk-grid-column-three-quarters-from-desktop">
             <span className="govuk-caption-l">API data set preview token</span>
             <h2>{dataSet?.title}</h2>
 
             {previewToken?.status === 'Active' ? (
               <>
+                <CopyTextButton
+                  buttonText="Copy preview token"
+                  confirmMessage="Preview token copied"
+                  id="copy-preview-token"
+                  label="Preview token"
+                  text={previewToken.id}
+                />
+
                 <p>
-                  Token expiry time:{' '}
+                  The token expires:{' '}
                   <strong>
                     <FormattedDate formatRelativeToNow>
                       {previewToken.expiry}
                     </FormattedDate>
                   </strong>
                 </p>
-                <h3>Using this token</h3>
-                <h4>Step 1</h4>
-                <p>TODO</p>
-                <h4>Step 2</h4>
-                <p>TODO else</p>
-                <h4>Final checks</h4>
-                <CopyTextButton
-                  buttonText="Copy preview token"
-                  className="govuk-!-margin-bottom-4"
-                  confirmMessage="Token copied to the clipboard"
-                  inlineButton={false}
-                  label="Preview token"
-                  text={previewToken.id}
-                />
+
+                <h3>Using the preview token</h3>
+
                 <p>
-                  Please revoke the token as soon as you have finished checking
-                  the API data set.
+                  To use the preview token, add it to your request using a{' '}
+                  <code>Preview-Token</code> header.
```

Comment:
I'm not sure about the use of the `<code>` here, as while this is referencing a code related property name it isn't actually code. The following section demonstrates how to use this header too so it may be redundant. My main concern is that there isn't a clear rule when to and when not to use this markup when referencing code variables, which may lead to some inconsistencies.

This is just my opinion, happy for you to take the steer on this just wanted to raise the potential inconsistency issue

Reply:
Think it's acceptable to use this here as we're referencing a technical term that needs to stand out.

We've only just added styling for `code` in the last couple of months so there's certainly going to be inconsistencies as we start to use it in more places. 

I think the general rule of thumb should be:

- is this a technical term?
- does it need to stand out to the user?

These are criteria that are being met a lot more by the public API work as it's much closer to the metal than other areas of the service.

Reply:
Fair point!